README.md

```diff
@@ -79,10 +79,10 @@ Submit pull requests [here](https://github.com/govCMS/dvf/pulls).
 
 ### Primary Developers
 
-[Doghouse Agency](https://www.drupal.org/doghouse-agency)
+[Doghouse Agency](http://doghouse.agency)
 
 ### Sponsors 
 
 * [govCMS](https://www.govcms.gov.au/)
-* [Dept of Environment](http://www.environment.gov.au/)
-* [Essential Services Commission](https://www.esc.vic.gov.au/)
+* [Department of Environment and Energy](http://www.environment.gov.au/)
```

Comment:
Technically if we're changing it, it's "Department of the Environment and Energy"

Reply:
Thanks Toby, good catch and updated now 👍

Reply:
hey - I can't call out much on the code side - so I'll play to my strengths!